discovery-handlers/onvif/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+pub mod discovery_handler;
+mod discovery_impl;
+mod discovery_utils;
+
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate yaserde_derive;
+
+/// Name that onvif discovery handlers use when registering with the Agent
+pub const DISCOVERY_HANDLER_NAME: &str = "onvif";
+/// Defines whether this discovery handler discovers local devices on nodes rather than ones visible to multiple nodes
```

Comment:
Since this is "shared" now, shouldn't that be true?
Same comment applies to the other DHs

Reply:
big oops on my part. Thank you for catching this. fixed in 207c1cc49ddd